docs/static/upgrading.asciidoc

```diff
@@ -106,87 +109,38 @@ Before upgrading Logstash:
+We recommend that you upgrade to {prev-major-last}, and then upgrade to 8.0.
 
-[float]
+[discrete]
 [[upgrade-to-6.8-rec]]
-==== Upgrade to {ls} 6.8 before upgrading to 7.0
+==== Upgrade to {ls} {prev-major-last} before upgrading to 8.0
 
-If you haven't already, upgrade to version 6.8 before you upgrade to 7.0. If
+If you haven't already, upgrade to version {prev-major-last} before you upgrade to 8.0. If
 you're using other products in the {stack}, upgrade {ls} as part of the
 {stack-ref}/upgrading-elastic-stack.html[{stack} upgrade process].
 
-TIP: Upgrading to {ls} 6.8 will give you a head-start on new 7.0 features, including
-the java execution engine and the strict field reference parser, while you're still running 6.x.
+TIP: Upgrading to {ls} {prev-major-last} gives you a head-start on new 8.0 features.
 This step helps reduce risk and makes roll backs easier if you hit
 a snag.
 
-//TO DO:  Add links [[field-ref-strict]] and [[java-exec-default]] after upgrade docs are merged
-
-Upgrading to 6.8 is required because the {es} index template was modified to
-be compatible with {es} 7.0 (the `_type` setting changed from `doc` to `_doc`).
-
-
 [[upgrading-logstash-pqs]]
-=== Upgrading with the Persistent Queue Enabled
-
-If you have the persistent queue (PQ) enabled, please read the section that applies
-for your upgrade scenario.
-
-* If you are upgrading from version 6.2.x or earlier, we recommend that you
-<<drain-pq,drain the persistent queue>> before you upgrade.
-
-* If you are upgrading from version 6.3.0 or later, see
-<<upgrading-logstash-pqs-6.3>> for information.
-
-[float]
-[[drain-pq]]
-==== Drain the Persistent Queue (version 6.2.x and earlier)
-
-The following applies only if you are upgrading from Logstash version 6.2.x or
-earlier with the persistent queue (PQ) enabled.
-
-We strive to maintain backward compatibility within a given major release. 
-Serialization issues in Logstash 6.2.x and earlier required us to break
-that compatibility in version 6.3.0 to ensure correctness of operation. For more
-technical details, please check our tracking github issue for this
-matter, https://github.com/elastic/logstash/issues/9494[#9494].
-
-We strongly recommend that you drain or delete
-the persistent queue before you upgrade from version 6.2.x and earlier.
-
-To drain the queue:
- 
-. In the logstash.yml file, set `queue.drain: true`.
-. Restart Logstash for this setting to take effect. 
-. Shutdown Logstash (using CTRL+C or SIGTERM), and wait for the queue to empty.
-
-When the queue is empty:
-
-. Complete the upgrade.
-. Restart Logstash.
-
-We have resolved issues with data incompatibilities for version 6.3 and later. 
-These steps won’t be required for future upgrades.
-
-[float]
-[[upgrading-logstash-pqs-6.3]]
-==== Upgrading from version 6.3 (and later) with Persistent Queues Enabled 
+=== Upgrading with the persistent queue enabled
```

Comment:
I removed a lot of PQ upgrade content specific to pre-6.3.  I left a smaller chunk of content for upgrading PQs. Let's decide if this makes sense or should be removed.

Reply:
I'm ok to remove, because in `7.x` series the PQ format didn't change so there isn't any path forward in "upgrading" the PQ. If a user has an older version he has however to move 6.x -> 7.x -> 8 so reading the upgrade path of previous major versions he is covered.